pkg/render/common/test/testing.go

```diff
@@ -253,3 +255,53 @@ func CreateCertSecret(name, namespace string, dnsNames ...string) *corev1.Secret
 		},
 	}
 }
+
+func ExpectBundleContents(bundle *corev1.ConfigMap, secrets ...types.NamespacedName) {
```

Comment:
One thing to note about the certificate bundle is that if you have CA A and A is the signer of B and C. Then if A is already in the bundle, B and C don't need to be added any longer. The Bundle implementation does contain this logic. This helper function is not aware of this and may result in wrong assertions. Therefore, I am not sure it belongs in a generic testing package.

Reply:
Yeah, that was something I was aware of - I think it's just up to the test writer to put the correct inputs to this function, and I think that's the way it should work - we shouldn't try to duplicate that logic as part of our test assertions.

Reply:
Personally I think it's totally appropriate in a generic testing package, and it does exactly what it says it does - it asserts that the bundle contents contain the expected values.